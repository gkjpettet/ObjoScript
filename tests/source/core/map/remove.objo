var map = {"a" : 0, "b" : 1, "c" : 2}
System.print(map.count) # expect 3

System.print(map.remove("A")) # expect nothing as case-sensitive
System.print(map.count) # expect 3

System.print(map.remove("a")) # expect 0
System.print(map.count) # expect 2