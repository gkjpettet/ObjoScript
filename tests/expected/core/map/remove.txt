3
nothing
3
0
2